beacon-chain/sync/pending_blocks_queue.go

```diff
@@ -35,10 +35,10 @@ func (s *Service) processPendingBlocksQueue() {
 	locker := new(sync.Mutex)
 	runutil.RunEvery(s.ctx, processPendingBlocksPeriod, func() {
 		locker.Lock()
+		defer locker.Unlock()
```

Comment:
Change this back

Reply:
I missed this. We shouldn't use defer there

Reply:
reverted